app/src/main/java/com/github/se/icebreakrr/ui/profile/OtherProfileView.kt

```diff
@@ -83,6 +93,27 @@ fun OtherProfileView(
               sendRequest = true
             }
             InfoSection(profile, tagsViewModel)
+
+            // Add spacer for some padding
+            Spacer(modifier = Modifier.height(16.dp))
+
+            // Already met button
+            Button(
+                onClick = {
+                  Toast.makeText(context, "Not yet implemented", Toast.LENGTH_SHORT).show()
```

Comment:
Same for the text in the toast. If we have a file containing all the constants for toasts you can put it there, otherwise you can put it at the beginning of your file.